src/mgr/DaemonServer.cc

```diff
@@ -2370,8 +2370,7 @@ void DaemonServer::adjust_pgs()
 		         << (is_merge_source ? " (merge source)" : " (merge target)")
 		         << dendl;
 	        ok = false;
-	      } else if (!(q->second.state & (PG_STATE_ACTIVE |
-					      PG_STATE_CLEAN))) {
+	      } else if (q->second.state != (PG_STATE_ACTIVE | PG_STATE_CLEAN)) {
```

Comment:
I think we actually want
```
(q->second.state & (PG_STATE_ACTIVE |
￼						      PG_STATE_CLEAN)) != (PG_STATE_ACTIVE|PG_STATE_CLEAN)
```
(other bits like scrubbing are fine)

Reply:
> other bits like scrubbing are fine

I forgot that, now fixed and repushed.